Harvest.Web/ClientApp/src/Expenses/ExpenseEntryContainer.tsx

```diff
@@ -98,7 +101,12 @@ export const ExpenseEntryContainer = () => {
     const response = await request;
 
     if (response.ok) {
-      history.push("/project");
+      // go to the project page unless you are a worker -- worker can't see the project page
+      if (roles.includes("Worker")) {
+        history.push("/");
```

Comment:
@srkirkland This goes to a react home?

![image](https://user-images.githubusercontent.com/469010/127902433-c303bfc9-e9ba-495e-8e0f-8c9ceccdd6a3.png)

Reply:
I didn't actually try it but why wouldn't it work?  what's the url?

Reply:
https://localhost:44308/

Reply:
If I refresh, the normal home page shows up.

Reply:
Ok i think i see it, we left in some debug code when doing the home route.  it'll take a look and see if that's an easy fix.

Reply:
Yea, we don't actually have a react homepage -- I'll make an issue for that and do it there.